server/@types/forms/index.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'forms' {
+  export interface CreateGoalForm {
+    prisonNumber: string
+    title: string
+    reviewDate: Date
+    steps: [
+      {
+        title: string
+        targetDate?: Date
+        sequenceNumber: number
```

Comment:
I guess we could make Step an object in its own right, but what would we call it? I don't think `CreateStepForm` would be right.